Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -473,7 +453,6 @@ static Color activeButtonTextColor()
         // The following colors would expose user appearance preferences to the web, and could be used for fingerprinting.
         // These are available only when the web view opts into the system appearance.
         case CSSValueWebkitFocusRingColor:
-        case CSSValueActiveborder:
```

Comment:
I think all the changes to this file should be made behind a linked-on-or-after check, to avoid potentially breaking older apps.

This part of the change would need to be done by someone at Apple, as there are other dependencies. Happy to take that part on, but I would need to make that a separate PR.

Reply:
I have reverted changes in this file and added failing `TestExpectation` for macOS only.